src/Uno.Toolkit.UI/Controls/NavigationBar/NativeFramePresenter.Android.cs

```diff
@@ -151,7 +155,7 @@ private async Task UpdateStack(PageStackEntry? entry, NavigationEventArgs e)
 					{
 						if (oldPage != null)
 							await oldPage.AnimateAsync(GetExitAnimation());
-						oldPage.ClearAnimation();
+						oldPage!.ClearAnimation();
```

Comment:
Are we missing braces here? ie, should `oldPage.ClearAnimation` be inside the `if`?